mscore/menus.cpp

```diff
@@ -1464,6 +1464,7 @@ void MuseScore::setDefaultPalette()
       paletteBox->addPalette(newFingeringPalette());
       paletteBox->addPalette(newRepeatsPalette());
       paletteBox->addPalette(newFretboardDiagramPalette());
+      paletteBox->addPalette(newAccordionPalette());
```

Comment:
Isn't it necessary also to add it to workspace files to actually add it to palettes? Or you didn't intend to do so?

Reply:
Actually I deliberatly didn't add it to any workspace, yet. And I'd rather remove the bagpipe embelshiments from the Advanced worksspace too.
And to be honest: I have no idea what that `setDefaultPalette()` does or is good for...

Reply:
What do you (plural) think, should we also remove the bagpipe emblishments from the advanced palette? That and this accordion palette are really seldom used, usefull for those that need them, but too much for the general users, even for advanced ones IMHO. And those uses that need them on a regular basis a) still/then have them available in the master plattetes and b) can put them into their custom workspace.

Reply:
As agreen on in the developers' chat I've now removed the bagpipe embellishments from the Advanced workspace